project/LinkerdBuild.scala

```diff
@@ -474,7 +488,7 @@ object LinkerdBuild extends Base {
     object Protocol {
 
       val h2 = projectDir("linkerd/protocol/h2")
-        .dependsOn(core, Router.h2, k8s, Finagle.h2 % "test->test;e2e->test")
+        .dependsOn(core, Router.h2, istio, Finagle.h2 % "test->test;e2e->test")
```

Comment:
I wonder if the istio identifiers should be in a separate project somewhere?  
istio-identifiers depends on [
  * linkerd-protocol-h2
  * linkerd-protocol-http
  * istio
]

Something like that?

Reply:
Yeah, I think I'm inclined to agree w/ @adleong on this – I feel like just replacing every dependency on `k8s` with `istio` doesn't _really_ accomplish the goal of this changeset (i.e. to decouple our Kubernetes and Istio modules)?

Reply:
This makes sense. Because this branch is a dependency for the mixer work I'll make an issue for it and tackle once that work is done